src/test/resources/org/rust/ide/annotator/fixtures/attributes.rs

```diff
@@ -1,4 +1,4 @@
 <info>#[cfg_attr(foo)]</info>
 fn <info>main</info>() {
-    <info>#![crate_type = "lib"]</info>
+    <info>#![crate_type = <info>"lib"</info>]</info>
```

Comment:
I don't know honestly, but I believe nested tags are supported; otherwise test should fail